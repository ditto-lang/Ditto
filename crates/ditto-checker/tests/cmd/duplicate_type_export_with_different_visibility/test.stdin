module Test exports (A, A(..));

type A = A;

-- NOTE: possibly deserves a different warning as the type visibility differs between the duplicates?
