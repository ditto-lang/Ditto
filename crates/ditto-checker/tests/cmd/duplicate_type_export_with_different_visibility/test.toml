bin.name = "ditto-checker-testbin-check-module"
args = ["duplicate_type_export_with_different_visibility.ditto"]
fs.sandbox = true
