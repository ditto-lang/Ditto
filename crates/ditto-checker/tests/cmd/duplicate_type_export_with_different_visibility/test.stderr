
  ⚠ duplicate type export
   ╭─[duplicate_type_export_with_different_visibility.ditto:1:1]
 1 │ module Test exports (A, A(..))
   ·                      ┬  ┬
   ·                      │  ╰── already exported
   ·                      ╰── previously exported here
 2 │ 
   ╰────

